### PR TITLE
Add issues/prs Assignments to the sidebar in the ui

### PR DESCRIPTION
**Pull request by cpanato**
will be like this:
screen shot 2018-08-10 at 16 26 53

Open question for future: Maybe we can have a way to configure what to show in the sidebar 😄

### CODE REVIEW

**Comment by cpanato** [comment]
It is hard to find good icons. if you have other suggestions I'm open for that

**Comment by crspeller** [comment]
Agree with Jason here. Seems like too many icons. Maybe don't distinguish between PR and issue assignments? Only one new icon?

**Comment by cpanato** [comment]
PR assignments shows in one page `pull/assigned` while the issue is in another page in github `issue/assigned`

agree that is too many, thats why I suggest to make the user to define which ones he wants.... 0/5

**Comment by jasonblais** [comment]
@cpanato This is the query for PRs and issues assigned to you https://github.com/pulls?q=is%3Aopen+archived%3Afalse+assignee%3Ajasonblais

**Comment by cpanato** [comment]
humm cool! i wil update the code 
thanks so much!

**Comment by cpanato** [comment]
done and update the PR @jasonblais

File: webapp/actions/index.js [hunk 1]
```diff
@@ -87,6 +87,31 @@ export function getYourPrs() {
...
+export function getYourAssignments() {
+    return async (dispatch, getState) => {
+        let data;
+        try {
+            data = await Client.getYourAssignments();
+        } catch (error) {
+            return {error};
+        }
+
+        let actions = [];
```
Comment (jwilander) [inline webapp/actions/index.js]: This variable looks unused

**Pull request merged**